### ISSUE TITLE
[ Feature request ] Rewards

### ISSUE THREAD

**Issue by BeReal86** [XGProyect/XG-Proyect-v3.x.x#436]
https://board.en.ogame.gameforge.com/index.php?thread/822141-23-11-reward-system-event-in-pts/

/game/index.php?page=ingame&component=rewarding

Tasks
![resim](https://user-images.githubusercontent.com/16218107/100031286-4bb88f00-2e06-11eb-8746-aa6fafba12e1.png)

Rank 1
![resim](https://user-images.githubusercontent.com/16218107/100031544-f16bfe00-2e06-11eb-8c4f-a7ff945a1204.png)

Rank 2
![resim](https://user-images.githubusercontent.com/16218107/100031558-fb8dfc80-2e06-11eb-9afe-f642fcca214c.png)

Rank 3
![resim](https://user-images.githubusercontent.com/16218107/100031571-034da100-2e07-11eb-98bd-8e95cec84273.png)

Rank 4
![resim](https://user-images.githubusercontent.com/16218107/100031594-12345380-2e07-11eb-9bcc-e5313215dbdf.png)

Rank 5
![resim](https://user-images.githubusercontent.com/16218107/100031613-1d877f00-2e07-11eb-904a-cae3307a47e3.png)

![resim](https://user-images.githubusercontent.com/16218107/100049517-8e8b5e80-2e28-11eb-8a53-a675208a85b1.png)

<img width="497" alt="Screenshot_2" src="https://user-images.githubusercontent.com/16218107/100090928-fbbce500-2e64-11eb-86da-3de735d2d6f3.png">

<img width="505" alt="Screenshot_8" src="https://user-images.githubusercontent.com/16218107/100344657-36a05380-2ff2-11eb-9570-7c3de4f78f90.png">

<img width="514" alt="Screenshot_9" src="https://user-images.githubusercontent.com/16218107/100344660-386a1700-2ff2-11eb-8e93-556e059bb1ce.png">

<img width="492" alt="Screenshot_12" src="https://user-images.githubusercontent.com/16218107/100491867-209cab00-3138-11eb-9f67-07692441fce9.png">

<img width="513" alt="Screenshot_1" src="https://user-images.githubusercontent.com/16218107/100516590-fe397a80-3195-11eb-88fe-a9fea0cbc3c1.png">

<img width="507" alt="Screenshot_2" src="https://user-images.githubusercontent.com/16218107/100540624-6d26da00-324f-11eb-93ec-d1ea4230a428.png">

**Comment by FGServers** [comment]
Dice Tritium sera un nuevo recurso?

**Comment by BeReal86** [comment]
No, it doesn't stand as a new resource right now. You earn him as his tasks are completed.

**Comment by LucasKovacs** [comment]
XGP4 is coming. XGP3 won't be supported anymore.
